openedx_webhooks/info.py

```diff
@@ -35,7 +42,50 @@ def _read_repotools_file(filename):
     return resp.text
 
 def get_people_file():
-    return _read_repotools_yaml_file("people.yaml")
+    """
+    Returns data formatted as a dictionary of people containing this information:
+    {
+        github_username: {
+            name: "",
+            agreement: "",
+            institution: ""
+            jira: "",
+            other_emails: [...],
+            committer: {...},
+            comments: [...],
+            before: {...},
+            ...
+        }
```

Comment:
This got put in the wrong place.  The comma-dot-dot-dot should be after the dict of github_username, to indicate that there are many usernames in the result.

Reply:
Yup, I added in the wrong place. Fixed!